src/storage/v2/CMakeLists.txt

```diff
@@ -42,4 +42,4 @@ add_library(mg-storage-v2 STATIC
         inmemory/replication/recovery.cpp
 )
 
-target_link_libraries(mg-storage-v2 mg::replication Threads::Threads mg-utils gflags absl::flat_hash_map mg-rpc mg-slk mg-events mg-memory)
+target_link_libraries(mg-storage-v2 mg::replication Threads::Threads mg-utils gflags absl::flat_hash_map mg-rpc mg-slk mg-events mg-memory mg-query)
```

Comment:
why that, this creates dependency between storage and query?

Reply:
To do periodic python GC we need PyCollectGarbage from PyModule, I don't know a way to avoid the dependancy

Reply:
ha but I think this could cause compilation slow-down... @Ignition advices?

Reply:
This highlights a design smell.

utils::Scheduler gc_python_runner_;
AND 
utils::Scheduler gc_jemalloc_runner_;

Should not be scoped to be the responsibility of storage. 
We need 1 runner for all of memgraph for jemalloc, hence should be in memgraph.cpp or some somewhere in glue code.
For python GC (not looked too deeply) either it needs to be 1 for all of memgraph (if we have a singleton python interpreter) or once per interpreter (which is would be query scoped).

Reply:
Moved both the jemalloc and python schedulers to memgraph.cpp so no more dependancy